src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -122,6 +122,16 @@ add_subdirectory(magnum-plugins EXCLUDE_FROM_ALL)
 add_dependencies(your-app MagnumPlugins::BasisImageConverter)
 @endcode
 
+When compiling Basis from source with `BASIS_UNIVERSAL_DIR`, Basis will be
+built with OpenCL support if the [OpenCL package](https://cmake.org/cmake/help/v3.1/module/FindOpenCL.html)
+is found. To build Basis without a hard dependency on OpenCL, do the following:
```

Comment:
I'd say this instead, as otherwise it might look like OpenCL is possible to disable only if you're having a CMake subproject.

```
pass `-DCMAKE_DISABLE_FIND_PACKAGE_OpenCL=ON` to CMake, or 
@cmake set() @ce it before the @cmake add_subdirectory() @ce call if 
using Magnum Plugins as a subproject.
```